.travis.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+language: node_js
```

Comment:
Why Node.js? The used language for this application is Vala

Reply:
Node.js is used to run a tool of the elementary project. The tool checks your app for compatibility.